### DISCUSSION TITLE
IBAX'ın geleceği ne olur?

### DISCUSSION THREAD

**Discussion by Modelactor** [IBAX-io/go-ibax#692] [Soru Cevap - Türkçe]
Bir avax gibi geleceği parlak mı yoksa daha mı iyi olur?

**Comment by Yusufknk** [comment]
İlerleyen süreçte bunu hep birlikte tecrübe edeceğiz.

**Comment by satocn** [comment]
Neden olmasın güzel yere gelebilir. Nerede önsatış yapılacağı çok önemli.
